CHANGELOG.md

```diff
@@ -1,3 +1,52 @@
+## [2.27.0] (2023-07-04)
+
+ENHANCEMENTS:
+
+* Introduce v1.5 `check` block ([terraform-schema#229](https://github.com/hashicorp/terraform-schema/pull/229))
+* Introduce v1.5 `import` block ([terraform-schema#228](https://github.com/hashicorp/terraform-schema/pull/228))
+* Re-generate function signatures for v1.5 ([terraform-schema#213](https://github.com/hashicorp/terraform-schema/pull/213))
+* Add v1.4 `local-exec` provisioner `quiet` attribute ([terraform-schema#218](https://github.com/hashicorp/terraform-schema/pull/218))
+* Reflect 1.4 changes in `gcs` backend ([terraform-schema#227](https://github.com/hashicorp/terraform-schema/pull/227))
+* Reflect 1.4 changes in `http` backend ([terraform-schema#226](https://github.com/hashicorp/terraform-schema/pull/226))
+* Reflect 1.3 & 1.4 changes in `cos` backend ([terraform-schema#217](https://github.com/hashicorp/terraform-schema/pull/217))
+* Reflect 1.2 & 1.3 changes in `azurerm` backend ([terraform-schema#225](https://github.com/hashicorp/terraform-schema/pull/225))
+
+INTERNAL:
+
+* Add copyright headers automagically instead of failing a check on PRs ([#1456](https://github.com/hashicorp/vscode-terraform/pull/1456))
+* Bump @vscode/test-electron from 2.3.0 to 2.3.2 ([#1429](https://github.com/hashicorp/vscode-terraform/pull/1429))
+* Bump @vscode/test-electron from 2.3.2 to 2.3.3 ([#1479](https://github.com/hashicorp/vscode-terraform/pull/1479))
+* Bump actions/checkout from 3.5.2 to 3.5.3 ([#1480](https://github.com/hashicorp/vscode-terraform/pull/1480))
+* Bump dessant/lock-threads from 4.0.0 to 4.0.1 ([#1485](https://github.com/hashicorp/vscode-terraform/pull/1485))
+* Bump eslint from 8.40.0 to 8.41.0 ([#1439](https://github.com/hashicorp/vscode-terraform/pull/1439))
+* Bump eslint from 8.41.0 to 8.42.0 ([#1465](https://github.com/hashicorp/vscode-terraform/pull/1465))
+* Bump eslint from 8.42.0 to 8.43.0 ([#1497](https://github.com/hashicorp/vscode-terraform/pull/1497))
```

Comment:
```suggestion
* Bump eslint from 8.40.0 to 8.43.0 ([#1439](https://github.com/hashicorp/vscode-terraform/pull/1439), [#1465](https://github.com/hashicorp/vscode-terraform/pull/1465), [#1497](https://github.com/hashicorp/vscode-terraform/pull/1497))
```

What do you think of combining all versions bumps for a single packet into a single line? We could also just link to the latest PR and skip the older ones:
```
* Bump eslint from 8.40.0 to 8.43.0 ([#1497](https://github.com/hashicorp/vscode-terraform/pull/1497))
```